jax/_src/config.py

```diff
@@ -1212,6 +1212,15 @@ def _update_jax_memories_thread_local(val):
           '  filesystem being used for the cache. '
           '* > 0: the actual minimum size desired; no overrides.'))
 
+# TODO: Change default to all when xla_gpu_kernel_cache_file issues are fixed.
```

Comment:
Update the comment to say to update the default later?

Reply:
Updated